### DISCUSSION TITLE
Vendure Integration with our project

### DISCUSSION THREAD

**Discussion by shubhamk-27** [vendurehq/vendure#1043] [Q&A]
1. We're building video commerce platform and currently exploring vendure as a possible solution.In our case a single video of merchant(channels) can display many products.How can we create a video relation with products in vendure? 

2. Our facets will not be channel specific, as we will be building a global category from which merchants can select their products . After V1.0 we came to know that facets and assets become channel specific. Does our case will fit in ?

3. We are going to use react-native-nodemediaclient for a video broadcasting purpose. 
Does it have any breaking changes when we will work with vendure.

**Comment by michaelbromley** [comment]
Hi! 

1. As an example, you can define a custom [Video entity](https://www.vendure.io/docs/plugins/plugin-examples/defining-db-entity/) and then create a [custom relation field](https://www.vendure.io/docs/developer-guide/customizing-models/#relations) on the Product entity to relate a Video to a Product. 
2. This should not be a problem, because channel-aware entities can be assigned to any number (or all) channels. So you can create some logic that will assign all Facets to a channel whenever a new one is created.
3. I have no idea about that package so I cannot answer definitively.
